README.md

```diff
@@ -26,7 +26,7 @@ The `autoload.php` file should contain rules for the package. For example:
 
 use Silverorange\Autoloader;
 
-$package = new Package('silverorange/site');
+$package = new Package(__DIR__);
```

Comment:
The package constructor has changed. Now each package is responsible for providing the base directory of any files it is responsible for. Previously the Autoloader was calculating the base directory from location of the `Autoloader.php`. This assumed that the `Autoloader.php` file was located within the vendor directory and wasn't under a symbolic link. This changes allows for us to use a development version of the `Silverorange_Autoloader`.